vpr/src/place/place.cpp

```diff
@@ -1466,7 +1497,26 @@ static e_move_result try_swap(float t,
         /* 1 -> move accepted, 0 -> rejected. */
         move_outcome = assess_swap(delta_c, t);
 
-        if (move_outcome == ACCEPTED) {
+        if (manual_move && manual_move_info->valid_input) {
+            //update all the costs in the manual_move_info variable and open cost summary window
+            manual_move_info->delta_c = delta_c;
+            manual_move_info->bb_delta_c = bb_delta_c;
+            manual_move_info->timing_delta_c = timing_delta_c;
+            if (move_outcome == REJECTED)
+                manual_move_info->placer_move_outcome = MOVE_REJECTED;
+            else if (move_outcome == ACCEPTED)
+                manual_move_info->placer_move_outcome = MOVE_ACCEPTED;
+            else
+                manual_move_info->placer_move_outcome = MOVE_ABORTED;
+            cost_summary_window();
+            update_screen(ScreenUpdatePriority::MAJOR, " ", PLACEMENT, nullptr);
+        }
+
+        if (manual_move && manual_move_info->valid_input) {
+            manual_move_outcome = (manual_move_info->user_move_outcome == MOVE_REJECTED ? REJECTED : ACCEPTED);
+        }
+
+        if ((!manual_move && move_outcome == ACCEPTED) || (manual_move && manual_move_outcome == ACCEPTED)) {
```

Comment:
Should be able to simplify this code by just having one move_outcome which is set by either manual moves or regular moves.

Reply:
resolved